src/extension.ts

```diff
@@ -30,6 +30,9 @@ import config from './config';
 import handleNewCljFiles from './fileHandler';
 import lsp from './lsp';
 import * as snippets from './custom-snippets';
+import * as docMirror from './doc-mirror'
+import { testDataDir } from './extension-test/integration/suite/util';
+import { commands } from 'fast-check/*';
```

Comment:
This looks to maybe have been auto imported or something / mistakenly left as well.